modules/Quiz/resources/quiz.js

```diff
@@ -936,6 +936,8 @@
             });
             _this.KIVQModule.submitAnswer(questionNr,_this.selectedAnswer);
             _this.selectedAnswer = null;
+            $(".cp-navigation").css("display", "none");
+            $(".ftr-container").css("height", "40px");
```

Comment:
@shalommeoded - we need to make sure this code does not interfere with all 4 question types 
If you have checked that - I can review

Reply:
Checked:
	
         1. On Multiple it works, just fine.
	2. On True and False as well, since it uses the same logic.
	3. Reflection point, not relevant.
	4. Same for open Q